### ISSUE TITLE
Cannot setup for latest Symfony5.2.x

### ISSUE THREAD

**Issue by SCIF** [getsentry/sentry-symfony#446]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io)

Which SDK and version?
Bundled

### Steps to Reproduce

1. Setup Symfony 5.2
2. `composer require sentry/sentry-symfony dev-master`

### Expected Result

Success installation

### Actual Result

```
Script cache:clear returned with error code 1
!!  
!!  In DefinitionErrorExceptionPass.php line 54:
!!                                                                                 
!!    Cannot autowire service "Symfony\Bridge\PsrHttpMessage\Factory\PsrHttpFacto  
!!    ry": argument "$serverRequestFactory" of method "__construct()" references   
!!    interface "Psr\Http\Message\ServerRequestFactoryInterface" but no such serv  
!!    ice exists. Did you create a class that implements this interface?           
!!                                                                                 
!!  
!!  
Script @auto-scripts was called via post-update-cmd
```

Quick digging gets me an idea of having prerequisite step `composer require nyholm/psr7`. Worked then.

**Comment by SCIF** [comment]
I think it's related to https://github.com/symfony/psr-http-message-bridge/releases/tag/v2.1.0

**Comment by ste93cry** [comment]
Albeit the error is slightly different, I think it's still related to #379. I've added a warning notice in the README about it a few days ago 😉

**Comment by SCIF** [comment]
@ste93cry , thanks. I found https://github.com/getsentry/sentry-symfony/tree/develop#optional-use-custom-http-factorytransport but it should be linked/referenced from/to https://github.com/getsentry/sentry-symfony/tree/develop#installation-with-symfony-flex-symfony-4-or-newer as well.

**Comment by ste93cry** [comment]
I'm not sure I understand what the customization of the HTTP factory/transport has to do with the issue you have, so I assume you wanted to write the link to the notice which is present in the `master` branch and not yet in `develop`

**Comment by SCIF** [comment]
@ste93cry , ah, I missed it as it was added to non-Flex installation instruction. But flex installations are affected as well

**Comment by ste93cry** [comment]
Right 😃 When I added it at the bottom of the install instructions the intention was not to mean that it was affecting just the non-Flex method, but it looks so because it has no proper section title. A PR to move the notice around so that it's either more prominent or more clear about it affecting both ways of installation is welcome. Not sure which solution is the best though, since giving it a big header imho is not appropriate as it's a warning and not a section of the README

**Comment by SCIF** [comment]
Looks like the installation process could be a little bit revised and made the single merged version:
0) Explanation about psr-7. Install something if you didn't.
1) `composer require sentry/sentry-symfony` // as common for  both ways
2.1) if you use `flex`, then just click `y`
2.2) otherwise, register the bundle  in the Kernel

That's just my humble opinion and not the proper solution :) Sorry, I'm not going to make a PR for that because of lack of knowledge what's going on and pretty generic english skills :)

**Comment by ste93cry** [comment]
I've opened a PR to improve the README. Give it a look if you want and let me know what you think 😉 In the meanwhile, I think we can close this, can't I?

**Comment by SCIF** [comment]
Fixed in #452
